### PR TITLE
German translation for July 2024 newsletter

### PR DESCRIPTION
**Pull request by Millesimus**
Hey folks,

as I wrote in the communications MUC I translated the most recent newsletter to get a feeling for the task. Now I'm submitting my translation to familiarise myself with the workflow.

I'd be glad if you could confirm / deny if this is how you would like to receive the translations.  It's no problem if this PR comes too late to be actually merged.

### CODE REVIEW

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+date: 2024-08-05
+categories: ['Newsletter']
+---
+
+{{< figure src="/images/newsletter/xmpp_newsletter_banner.jpg" caption="XMPP Newsletter Banner" >}}
+
+Herzlich willkommen zum XMPP Newsletter! Schön, dass Du hier bist.
+Diese Ausgabe behandelt Juli 2024.
+
+##  Meldungen der XSF
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:12]:
```suggestion
## Meldungen der XSF
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+Wenn Du Mitglied der XMPP Standards Foundation (XSF)  werden möchtest, [kannst Du Dich noch bis einschließlich 18. August 2024 bewerben](https://wiki.xmpp.org/web/Membership_Applications_Q3_2024)!
+  
+### XMPP beim Google Summer of Code 2024
+
+Die XSF wurde auch 2024 wieder als [Gastgeber-Organisation im Google Summer of Code](https://wiki.xmpp.org/web/Google_Summer_of_Code_2024)! Folgende XMPP-Projekte haben Plätze erhalten und sind am coden:
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:18]:
```suggestion
Die XSF wurde auch 2024 wieder als [Gastgeber-Organisation im Google Summer of Code](https://wiki.xmpp.org/web/Google_Summer_of_Code_2024)! Folgende XMPP-Projekte haben Plätze erhalten und sind am Coden:
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+- [Monal](https://monal-im.org/)
+  - [Modernes Onboarding und Mediengallerie in Monal] (175 Arbeitsstunden, Medium)](https://wiki.xmpp.org/web/Gsoc2024/Monal/Media_Gallery)
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:21]:
```suggestion
  - [Modernes Onboarding und Mediengalerie in Monal] (175 Arbeitsstunden, Medium)](https://wiki.xmpp.org/web/Gsoc2024/Monal/Media_Gallery)
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+    - [1. Blogpost](https://thevaidik.medium.com/google-summer-of-code-gsoc-my-experience-1-xmpp-standards-foundation-da781ac95560)
+    - [2. Blogpost](https://thevaidik.medium.com/google-summer-of-code-gsoc-my-experience-2-midterm-evaluations-xmpp-standards-foundation-3be8b27dc653)
+- [Prav.app](https://prav.app/)
+  - INFO: Unglücklicherweise hat das Projekt die Zwischenevaluation nicht bestanden und wurde eingestellt.
+  - [Standard-konforme SMS-Authentifizierung via OTP compliant] (350 Arbeitsstunden, Medium)](https://wiki.xmpp.org/web/Gsoc2024/Prav.app/Standards_compliant_SMS_OTP_based_authentication)
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:26]:
```suggestion
  - [Standard-konforme SMS-Authentifizierung via OTP] (350 Arbeitsstunden, Medium)](https://wiki.xmpp.org/web/Gsoc2024/Prav.app/Standards_compliant_SMS_OTP_based_authentication)
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+    - [1. Blogpost](https://write.as/assu/gsoc-and-open-source-my-first-steps-into-xmpp)
+
+{{< figure src="/images/logos/gsoc2024_logo.svg" caption="XSF and Google Summer of Code 2024" >}}
+
+### Spenden für XMPP-Projekte
+
+Die XSF bietet XMPP-Projekten an, die [Verwaltung ihrer Finanzen](https://xmpp.org/community/fiscalhost/) zu übernehmen. Projekte können dies via via [Open Collective](https://opencollective.com/xmpp) anfragen. Für weitere Informationen, siehe [die ursprüngliche Ankündigung im Blog](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Derzeit kann so an folgende Projekte gespendet werden:
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:33]:
```suggestion
Die XSF bietet XMPP-Projekten an, die [steuerliche Verwaltung ihrer Finanzen](https://xmpp.org/community/fiscalhost/) zu übernehmen. Projekte können dies via [Open Collective](https://opencollective.com/xmpp) anfragen. Für weitere Informationen, siehe [die ursprüngliche Ankündigung im Blog](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Derzeit kann so an folgende Projekte gespendet werden:
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+## XMPP-Veranstaltungen
+
+- [XMPP Sprint in Worcester, UK](https://wiki.xmpp.org/web/Sprints/2024-09_Worcester_UK):  21. bis 22. September — The Hive, Worcester
+- [XMPP Track auf der FOSSY](https://2024.fossy.us/): 01. bis 04. August— Portland State University
+- [Berlin XMPP Meetup](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup)[DE / EN]: Monatliches Treffen von XMPP-Enthusiasten, Berlin, jeden zweiten Mittwoch im Monat
+- [Italienische XMPP Happy Hour](https://video.xmpp-it.net/c/happyhour/videos) [IT]: Monatliches, italienisches XMPP Online-Meeting, jeden dritten Montag im Monat um 19:00 Uhr Ortszeit
+
+## Artikel über XMPP
+
+- [Rivista](https://git.xmpp-it.net/sch/Rivista) ist ein neues Veröffentlichungssystem das aus XMPP PubSub-Knoten dynamisch eine Journal-Seite generiet. Rivista ist mit Atom/RSS kompatibel und kann auch im Browser gelesen werden.
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:49]:
```suggestion
- [Rivista](https://git.xmpp-it.net/sch/Rivista) ist ein neues Veröffentlichungssystem das aus XMPP PubSub-Knoten dynamisch eine Journal-Seite erstellt. Rivista ist mit Atom/RSS kompatibel und kann auch im Browser gelesen werden.
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+- [Ein Blogpost von Haiku-OS über den XMPP Sprint in Berlin im Juli 2024 (englisch)](https://www.haiku-os.org/blog/nephele/2024-07-21_xmpp_coding_sprint_berlin). Die [Liste der dort angegangenen Projekte](https://wiki.xmpp.org/web/Sprints/2024-07_Berlin#Attendees)  findet ihr im XMPP-Wiki.
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:50]:
```suggestion
- [Ein Blogpost von Haiku-OS über den XMPP Sprint in Berlin im Juli 2024 (englisch)](https://www.haiku-os.org/blog/nephele/2024-07-21_xmpp_coding_sprint_berlin). Die [Liste der dort bearbeiteten Projekte](https://wiki.xmpp.org/web/Sprints/2024-07_Berlin#Attendees) findet ihr im XMPP-Wiki.
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+- [JMP.chat-Newsletter Juli 2024: Anrufe aus SIP und ein möglicher neuer SMS-Tarif (englisch)](https://blog.jmp.chat/b/july-newsletter-2024)
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:51]:
```suggestion
- [JMP.chat-Newsletter Juli 2024: Anrufe aus SIP und ein möglicher neuer SIM-Tarif (englisch)](https://blog.jmp.chat/b/july-newsletter-2024)
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+- [ProcessOne: Kostenaufschlüsselung für große IM-Dienste (englisch)](https://www.process-one.net/blog/breaking-down-the-costs-of-large-messaging-services/)
+
+## Neuigkeiten über XMPP-Software
+
+### XMPP-Clients und -Anwendungen
+
+- [Rivista 0.1](https://git.xmpp-it.net/sch/Rivista) wurde veröffentlicht.
+- [Psi+ 1.5.2033 portable](https://sourceforge.net/projects/psiplus/files/Windows/Personal-Builds/tehnick/) wurde veröffentlicht
+- [Psi+ 1.5.2029 bis 1.5 2038 installer](https://sourceforge.net/projects/psiplus/files/Windows/Personal-Builds/KukuRuzo/) wurden veröffentlicht.
+- [Gajim 1.9.2](https://gajim.org/post/2024-07-19-gajim-1.9.2-released/) wurde veröffentlich. Die Änderungen umfasst eine wichtige Korrektur in der OMEMO-Verschlüsselung, native Benachrichtigungen unter Windows und Verbesserung in der Bedienung. Außerdem wurde [Gajim 1.9.3](https://gajim.org/post/2024-07-29-gajim-1.9.3-released/) mit einer Korrektur für den MS Store und weiteren Verbesserungen veröffentlicht.
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:61]:
```suggestion
- [Gajim 1.9.2](https://gajim.org/post/2024-07-19-gajim-1.9.2-released/) wurde veröffentlicht. Die Änderungen umfasst eine wichtige Korrektur in der OMEMO-Verschlüsselung, native Benachrichtigungen unter Windows und Verbesserungen in der Bedienung. Außerdem wurde [Gajim 1.9.3](https://gajim.org/post/2024-07-29-gajim-1.9.3-released/) mit einer Korrektur für den MS Store und weiteren Verbesserungen veröffentlicht.
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+- [Monal 6.4.1](https://github.com/monal-im/Monal/releases/tag/Build_iOS_941)wurde mit einigen Korrekturen und aktualisierten Übersetzungen veröffentlicht. ACHTUNG, das ist das letzte Release mit Unterstützung für MacOS 11 und 12 sowie iOS 14 und 15. Außerdem wurde [Monal IM](https://monal-im.org/) von der [NGI der EU](https://ngi.eu) für [eine weitere Förderphase](https://nlnet.nl/project/Monal-IM-UI/) des [NLnet Foundation NGI0 Entrust Fund](https://nlnet.nl/entrust/) ausgewählt, um an einigen wichtigen Features zu arbeiten. Namentlich: die Implementierung eines Wählfelds, die Überarbeitung des UI-Codes, die Implementierung von Reaktionen, klickbaren Antworten und Stickern, Arbeiten im Rahmen der XSF und die Dokumentation von Monals Software-Architektur. Weitere Details im[Monal IM Blogpost (englisch)](https://monal-im.org/post/00013-nlnet-funding2/).
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:62]:
```suggestion
- [Monal 6.4.1](https://github.com/monal-im/Monal/releases/tag/Build_iOS_941) wurde mit einigen Korrekturen und aktualisierten Übersetzungen veröffentlicht. ACHTUNG, das ist das letzte Release mit Unterstützung für MacOS 11 und 12 sowie iOS 14 und 15. Außerdem wurde [Monal IM](https://monal-im.org/) von der [NGI der EU](https://ngi.eu) für [eine weitere Förderphase](https://nlnet.nl/project/Monal-IM-UI/) des [NLnet Foundation NGI0 Entrust Fund](https://nlnet.nl/entrust/) ausgewählt, um an einigen wichtigen Features zu arbeiten. Namentlich: die Implementierung eines Wählfelds, die Überarbeitung des UI-Codes, die Implementierung von Reaktionen, klickbaren Antworten und Stickern, Arbeiten im Rahmen der XSF und die Dokumentation von Monals Software-Architektur. Weitere Details im [Monal IM Blogpost (englisch)](https://monal-im.org/post/00013-nlnet-funding2/).
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 11]
```diff
@@ -0,0 +1,189 @@
...
+- [Movim 0.26](https://github.com/movim/movim/releases/tag/v0.26) wurde veröffentlicht. Neue Features sind benutzerdefinierte Emojis, Codeblöcke in Nachrichten, besseres Spam-Handling, bessere Nachrichtenmoderation und -Rückzug  und natürlich Fehlerkorrekturen.  Darüber hinaus wurde[Movim](http://movim.eu/) von der [NLnet Foundation](https://nlnet.nl/) für Förderungen aus dem [NGI0 Core](https://nlnet.nl/core/) ausgewählt, um an [vielen Features](https://mov.im/node/pubsub.movim.eu/Movim/f619c49c-ccd3-4487-8c2e-20b6d3555da2) rund um  [Video-Konferenzen](https://nlnet.nl/project/Movim-E2EE-video/) zu arbeiten, unter Anderem one-to-many Audio- und Videoanrufeon the platform. Weitere Einblicke und technische Details folgen in künftigen Blogposts.
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:63]:
```suggestion
- [Movim 0.26](https://github.com/movim/movim/releases/tag/v0.26) wurde veröffentlicht. Neue Features sind benutzerdefinierte Emojis, Codeblöcke in Nachrichten, besseres Spam-Handling, bessere Nachrichtenmoderation und -Rückzug und natürlich Fehlerkorrekturen.  Darüber hinaus wurde[Movim](http://movim.eu/) von der [NLnet Foundation](https://nlnet.nl/) für Förderungen aus dem [NGI0 Core](https://nlnet.nl/core/) ausgewählt, um an [vielen Features](https://mov.im/node/pubsub.movim.eu/Movim/f619c49c-ccd3-4487-8c2e-20b6d3555da2) rund um [Video-Konferenzen](https://nlnet.nl/project/Movim-E2EE-video/) zu arbeiten, unter Anderem Gruppenanrufe mit Audio und Video. Weitere Einblicke und technische Details folgen in künftigen Blogposts.
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 12]
```diff
@@ -0,0 +1,189 @@
...
+
+## Erweiterumgen und Spezifikationen
+
+Die XSF entwickelt Erweiterungen für das XMPP-Protokoll. Diese [XEPs](https://xmpp.org/extensions/) ergänzen die ursprünglichen [XMPP-RFCs](https://xmpp.org/rfcs/).
+
+Entwickler:innen und andere Expert:innen für Standardisierung aus der ganzen Welt arbeiten für diese Erweiterungen zusammen. Sie entwickeln neue Spezifikationen für entstehende Features und Praktiken und verbessern bestehende Erweiterungen des Protokolls. Erweiterungen können von jedem Menschen vorgeschlagen werden. Falls erfolgreich, werden sie je nach Typ als _Final_ oder _Active_ hinterlegt. Andere werden sorgsam als _Deferred_ für die Zukunft bewahrt.
+
+Beschrieben wird dieser XEP-Lebenszyklus samt förmlicher und kanonischer Definitionen der Typen, Zustände und Prozesse im ersten XEP, [XEP-0001](https://xmpp.org/extensions/xep-0001.html). [Lerne mehr über den Standardisierungsprozess](https://xmpp.org/about/standards-process.html)
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:84]:
```suggestion
Beschrieben wird dieser XEP-Lebenszyklus samt förmlicher und kanonischer Definitionen der Typen, Zustände und Prozesse im ersten XEP, [XEP-0001](https://xmpp.org/extensions/xep-0001.html). [Lerne mehr über den Standardisierungsprozess](https://xmpp.org/about/standards-process.html).
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 13]
```diff
@@ -0,0 +1,189 @@
...
+  - Hinzufügen eines XML-Schemas.
+  - Hinweis auf das Hinzufügen eines neuen Namensraum, der beim Registrar eingereicht werden muss.
+  - Korrektur von Einrückungen, Tippfehlern, fehlerhafte Verwendung von `''` vs. `</>` für Elemente usw. (egp)
+- Version 0.1.2 des XEP-0491 (WebXDC)
+  - Vorschlag für selfAddr-Werte
+  - Hinzufügen von Danksagungen (spw)
+
+### Last Call (letzter Aufruf)
+
+Letzte Aufrufe (_Last Calls_) erfolgen sobald sich der Status eines XEPs konsolidiert hat. Heißt: Nachdem der Rat einen XEP für fertig befunden hat, startet die:er XEP-Editor:in einen letzten Aufruf für Kommentare. Die so eingehalten Rückmeldungen helfen, den XEP zu verbessern. Danach geht er zurück an den Rat, um als _Stable_ eingestuft zu werden.
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:122]:
```suggestion
Letzte Aufrufe (_Last Calls_) erfolgen sobald sich der Status eines XEPs konsolidiert hat. Heißt: Nachdem der Rat einen XEP für fertig befunden hat, startet die:er XEP-Editor:in einen letzten Aufruf für Kommentare. Die so erhaltenen Rückmeldungen helfen, den XEP zu verbessern. Danach geht er zurück an den Rat, um als _Stable_ eingestuft zu werden.
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 14]
```diff
@@ -0,0 +1,189 @@
...
+- [LinkedIn](https://www.linkedin.com/company/xmpp-standards-foundation/)
+- [Lemmy instance (inoffiziell)](https://slrpnk.net/c/xmpp)
+- [Reddit (inoffiziell)](https://www.reddit.com/r/xmpp/)
+- [XMPP Facebook page (inoffiziell)](https://www.facebook.com/jabber)
+
+{{< newsletter-subscribe >}}
+
+Du kannst den Newsletter auch als [RSS Feed](https://xmpp.org/feeds/all.atom.xml) abonnieren!
+
+Auf der Suche nach Jobangeboten im XMPP-Umfeld oder professionellem Consulting für ein XMPP-Projekt? Wirf einen Blick auf unser  [job board](https://xmpp.work/)!
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:154]:
```suggestion
Auf der Suche nach Jobangeboten im XMPP-Umfeld oder professionellem Consulting für ein XMPP-Projekt? Wirf einen Blick auf unser [job board](https://xmpp.work/)!
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 15]
```diff
@@ -0,0 +1,189 @@
...
+- Französisch: [jabberfr.org](https://news.jabberfr.org/category/newsletter/) und [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+  - Übersetzung: *Adrien Bourmault (neox), alkino, anubis, Arkem, Benoît Sibaud, mathieui, nyco, Pierre Jarillon, Ppjet6, Ysabeau*
+- Italienisch: [notes.nicfab.eu](https://notes.nicfab.eu)
+  - Übersetzung: *nicola*
+- Spanisch: [xmpp.org](https://xmpp.org/categories/newsletter/])
+  - Übersetzung: *Gonzalo Raúl Nemmi*
+
+## Hilf dem Newsletter!
+
+Der XMPP-Newsletter wird gemeinsam von der XMPP-Gemeinschaft geschrieben. Die monatlich erscheinende Ausgabe wird in einem [schlichten Pad](https://pad.nixnet.services/oHnY_ZvLT8SoFyCqIC2ung) vorbereitet und am Ende des Monats in das  [Github-Repositorium der XSF](https://github.com/xsf/xmpp.org) übertragen. Mach gerne mit! Komm in [unseren Chatraum (MUC)](xmpp:commteam@muc.xmpp.org?join) und hilf uns dabei, dieses Projekt fortzuführen. Hast Du ein neues Projekt, von dem Leute wissen sollten? Teile es in unserem Newsletter mit einem großen Publikum!
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:171]:
```suggestion
Der XMPP-Newsletter wird gemeinsam von der XMPP-Gemeinschaft geschrieben. Die monatlich erscheinende Ausgabe wird in einem [schlichten Pad](https://pad.nixnet.services/oHnY_ZvLT8SoFyCqIC2ung) vorbereitet und am Ende des Monats in das [Github-Repositorium der XSF](https://github.com/xsf/xmpp.org) übertragen. Mach gerne mit! Komm in [unseren Chatraum (MUC)](xmpp:commteam@muc.xmpp.org?join) und hilf uns dabei, dieses Projekt fortzuführen. Hast Du ein neues Projekt, von dem Leute wissen sollten? Teile es in unserem Newsletter mit einem großen Publikum!
```

File: content/blog/newsletter/2024-08-05-newsletter.de.md [hunk 16]
```diff
@@ -0,0 +1,189 @@
...
+- Übersetzungen
+- Kommunikation in den diversen sozialen Medien
+
+## Den Newsletter de-abonnieren
+Um Dich aus dieser Liste auszutragen, [logge Dich ein](https://mail.jabber.org/accounts/login/?next=/postorius/lists/newsletter.xmpp.org/).
+Wenn Du Dich noch nie eingeloggt hast, musst Du gegebenenfalls erst einen Account mit der entsprechenden E-Mail-Adresse erstellen.
+
+## Lizenz
+
+Dieser Newsletter wird unter der  [CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) veröffentlicht.
```
Comment (cal0pteryx) [inline content/blog/newsletter/2024-08-05-newsletter.de.md:189]:
```suggestion
Dieser Newsletter wird unter der [CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) veröffentlicht.
```

**Review by cal0pteryx — CHANGES_REQUESTED** [review]
Thanks a lot! I left some suggestions. If you are ok with all of those, I would merge this PR.

Regarding XEP changelogs, we came to the conclusion that it does not really make sense to translate those, since they are for technical people anyway. So the changelog can be kept in its original form in the next newsletter issue.

I like the translation for various XEP processes though!

**Comment by Millesimus** [comment]
Hey @cal0pteryx, thank you very much for your thorough and laborious review! I committed all your changes (though I realised too late that I wasn't adding them to the batch).

**Comment by cal0pteryx** [comment]
Thanks! :)

**Pull request merged**